styles/advantages.css

```diff
@@ -0,0 +1,51 @@
+.advantages__list {
+  display: flex;
+  justify-content: space-between;
+}
+
+.advantages__title {
+  margin-bottom: 28px;
+}
+
+ol {
```

Comment:
I assume this should be class selector, not global one.

Reply:
fixed